ui/app/utils/classes/stream-logger.js

```diff
@@ -73,5 +73,16 @@ export default EmberObject.extend(AbstractLogger, {
     }
   }),
 }).reopenClass({
-  isSupported: !!window.ReadableStream,
+  isSupported: !!window.ReadableStream && !isSafari(),
 });
+
+// Fetch streaming doesn't work in Safari yet despite all the primitives being in place.
+// Bug: https://bugs.webkit.org/show_bug.cgi?id=185924
```

Comment:
Is there any way we could completely replicate/use the environment we are seeing the bug, and then put your working test in it to see if it still works?

Reply:
By working test do you mean this branch? As in test this branch via https in Safari to see if this fixes the problem?

Reply:
Well it looks like you've reduced the code to something that someone with no knowledge of the nomad-ui can look at, which is super helpful, but that works - so theres no bug there.

If we can see if we can get this on the same environment of where your bug is. That way we can reproduce everything as much as possible but with simpler code, then and try to drill down on exactly what the problem is - I think you mention it could be something to do with differences with the certs? I think the nomad demo site cert uses SNI for example? I doubt its that but would be good if we can check - just ruling things out really.

Reply:
Ah, to replicate the Safari bug. Yes, that would be great, and as you saw I went down that road.

After failing to replicate it, I had to pull myself out of the rabbit hole and focus back on Nomad 😄

I did publish that code though if you wanted to continue exploring it: https://github.com/DingoEatingFuzz/safari-streaming

I hosted it in my DO droplet with a basic Let's Encrypt cert.